API.md

```diff
@@ -1212,6 +1213,41 @@ Example:
 
 Platforms: all
 
+#### onReadyForDisplay
+Callback function that is called when the first video frame is ready for display. This is when the poster is removed.
+
+Payload: none
+
+* iOS: [readyForDisplay](https://developer.apple.com/documentation/avkit/avplayerviewcontroller/1615830-readyfordisplay?language=objc)
+* Android [STATE_READY](https://exoplayer.dev/doc/reference/com/google/android/exoplayer2/Player.html#STATE_READY)
+
+Platforms: Android, iOS, Web
+
+#### onReceiveAdEvent
+Callback function that is called when an AdEvent is received from the IMA's SDK.
+
+Payload:
+
+| Property | Type   | Description           |
+|----------|--------|-----------------------|
+|  event   | string | The ad event received |
+
+Example:
+```
+{
+  "event": "LOADED"
```

Comment:
can you retail all types of ads events available please ?

Reply:
Will do!

Reply:
Done in the last commit